.github/workflows/osv-scanner.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Copyright (c) 2025, Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: OSV-Scanner
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches: ["master"]
```

Comment:
I think we need stable branches as well here.
Ignoring paths is also recommended. 
I would even skip the cron autostart, maybe.


The branches can be taken from here:

https://github.com/lf-edge/eve/blob/2c5c47d4fea30ff32b3dcaf54bf50324b7390993/.github/workflows/codeql.yml#L9-L27